utility/src/main/java/org/oppia/android/util/profile/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+"""
+General purposes utilities to manage Directories.
```

Comment:
```suggestion
General purposes utilities to manage directories.
```

No need to capitalize here.

Reply:
@BenHenning Ok I removed the capitalization.